lib/index.js

```diff
@@ -72,7 +75,7 @@ module.exports = {
     const upload = (file, customParams = {}) =>
       new Promise((resolve, reject) => {
         // upload file on S3 bucket
-        const { Key } = getPathKey(file);
+        const { Key } = getPathKey(file, config.upload);
```

Comment:
Hey, thanks for your PR!

In the config file above you are specifing `config.pool` but here you are using `config.upload`. Could it be that you have switched around the variables here? Can you please double check this.

Reply:
Totally true, sorry for this confusion. Now it is solved 😄.